test/JDBC/input/BABEL-3268.sql

```diff
@@ -0,0 +1,84 @@
+-- setup --
```

Comment:
Let's remove this file as `BABEL-3268-vu-XXX` tests are executed in both normal and upgrade tests.